ssl.go

```diff
@@ -42,3 +47,19 @@ func (c *API) ReprioSSL() {
 // 	DELETE /zones/:zone_identifier/custom_certificates/:identifier
 func (c *API) DeleteSSL() {
 }
+
+// SSLVerification gets the Domain Control Validation records customers
+// need to validate ownership of a domain for Universal SSL
+//  GET /zones/:zone_identifier/ssl/verification"
+func (c *API) SSLVerification(zoneID string) (SSLVerificationResponse, error) {
```

Comment:
Please make the method receiver signature `api *API` - we're making this consistent everywhere now. Early attempts were a little mixed. :-)